metadata/src/main/resources/common/metadata/TopicRecord.json

```diff
@@ -17,12 +17,16 @@
   "apiKey": 2,
   "type": "metadata",
   "name": "TopicRecord",
-  "validVersions": "0",
+  "validVersions": "0-1",
```

Comment:
Please add a comment above to mention what's the diff between v0 and v1. Like [this](https://github.com/apache/kafka/blob/trunk/metadata/src/main/resources/common/metadata/PartitionChangeRecord.json#L21).

Reply:
@showuon thanks for the review. Updated.

Reply:
I have a feeling that we will require to either a) mark this version as unstable while development is going on or b) introduce a new IBP. I was facing problem with this in the implementation of KIP-1005. I would suggest going for the unstable version and worrying about the IBP later on

Reply:
@clolov how to mark a version as unstable

Reply:
I see `"latestVersionUnstable": true`, cannot be applied on TopicRecord.json as the type is metadata, and it can be applied only on Request type.